aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,18 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## September 6 - September 17, 2021
+- Enhanced List to support vertical alignment and added guidance for [multi-line lists](/templates/lists).
+- Advanced Design System site search capabilities. Work in progress.
+- Aligned Toolbar Button hover implementation with [Figma](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=681%3A5164).
+- Completed research on [task focused navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A18).
+- Added guidance for [toggle icon buttons](/components/button#toggle-buttons-with-icons).
+- Completed a usability test on toggle buttons and sortable columns within a table utilizing a user who uses a screen reader.
+- Enhanced a11yText on our light/dark mode button on our [HPE Design System home page](https://design-system.hpe.design/).
+- [Ideation on Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1279%3A1206) coming to a finish.
+- Completed adding t-shirt sizing for buttons. Figma publishing coming soon!
```

Comment:
Can we just say `Added t-shirt sizing for buttons. Figma publishing coming soon!

Reply:
Updated in latest commit